docs/DeveloperGuide.md

```diff
@@ -1,34 +1,315 @@
 # Developer Guide
 
-## Design & implementation
+## Table of Contents
```

Comment:
The numberings of the content page do not line up
![image](https://user-images.githubusercontent.com/60381634/113231099-0da80c00-92cd-11eb-9235-22afd20350c0.png)
Additionally, the formatting of the sub sections are not standardized

Reply:
Numbering in the sections themselves are ok.